OiCampinas/Model/LearnerModel.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  LearnerModel.swift
+//  OiCampinas
+//
+//  Created by zein rezky chandra on 15/10/19.
+//  Copyright © 2019 Zein Rezky Chandra. All rights reserved.
+//
+
+import Foundation
+
+struct LearnerModel {
+    var learningObjective = 0
+    var mentorName = ""
+    var groupNumber = 0
+    var currentProjectName = ""
+    
+    init(learning_objective: Int, mentor_name: String, group_number: Int, current_project_name: String) {
```

Comment:
Why you have to put Init on struct zein??? 😒, it’s a struct not a class zein

Reply:
What do you mean? Can you give explanation why please? @davigun